packages/ui-bindings/agent/agent.tsx

```diff
@@ -51,12 +51,12 @@ query($token: String, $agentId: Int) {
         unplannedEconomicEvents {
           action
         }
-        processEconomicEvents {
-          action
-        }
-        processCommitments {
-          action
-        }
+        #processEconomicEvents {
```

Comment:
Why did you comment this out?

Reply:
The API changed. After I pulled the latest from api-extensions upstream, the naming had changed